Library/OcMainLib/OpenCoreUefiAudio.c

```diff
@@ -325,77 +147,40 @@ OcAudioAcquireFile (
   )
 {
   EFI_STATUS          Status;
-  CHAR8               TmpPath[8];
   OC_STORAGE_CONTEXT  *Storage;
-  CONST CHAR8         *BaseType;
-  CONST CHAR8         *BasePath;
   UINT8               *FileBuffer;
   UINT32              FileBufferSize;
-  BOOLEAN             Localised;
-  OC_AUDIO_FILE       *CacheFile;
-
-  Storage   = (OC_STORAGE_CONTEXT *)Context;
-  CacheFile = NULL;
-
-  if ((File >= OcVoiceOverAudioFileBase) && (File < OcVoiceOverAudioFileMax)) {
-    if (mEnableAudioCaching) {
-      CacheFile = &mOcAudioFiles[File - OcVoiceOverAudioFileBase];
-      if (CacheFile->Buffer != NULL) {
-        *Buffer     = CacheFile->Buffer;
-        *BufferSize = CacheFile->Size;
-        return EFI_SUCCESS;
-      }
-    }
-  } else if (File < AppleVoiceOverAudioFileMax) {
-    if (mEnableAudioCaching) {
-      CacheFile = &mAppleAudioFiles[File];
-      if (CacheFile->Buffer != NULL) {
-        *Buffer     = CacheFile->Buffer;
-        *BufferSize = CacheFile->Size;
-        return EFI_SUCCESS;
-      }
-    }
-  } else {
-    DEBUG ((DEBUG_INFO, "OC: Invalid wave index %d\n", File));
-    return EFI_NOT_FOUND;
-  }
-
-  BasePath = OcAudioGetFilePath (
-               File,
-               TmpPath,
-               sizeof (TmpPath),
-               &BaseType,
-               &Localised
-               );
 
-  if (BasePath == NULL) {
-    DEBUG ((DEBUG_INFO, "OC: Unknown Wave %d\n", File));
-    return EFI_NOT_FOUND;
+  if ((BasePath == NULL) || (BaseType == NULL) || (Buffer == NULL) || (*Buffer == NULL)) {
```

Comment:
I think we do not overuse braces in such situations.

Reply:
@vit9696 - uncrustify is currently requiring these, sorry, so left them in!